### DISCUSSION TITLE
Updating routers / and Groups

### DISCUSSION THREAD

**Discussion by TerminalAddict** [eduardogsilva/routerfleet#38] [General]
Hi guys,
first time user here.
the description says "offers simplified backups, updates, and security" 
How do I update my Mikrotik routers?

Secondly, I've added some groups, how do I add routers into groups?

**Comment by eduardogsilva** [comment]
Hello @TerminalAddict. Welcome to routerfleet.

Updating routers are still on the roadmap but not implemented yet. I'm working hard on hotspot beacon project and couldn't return to routerfleet. I plan to take a look soon.
I will implement this feature in hotspot beacon and also bring it to routerfleet.

To add routers on your groups, just edit the router group and  you will see a "Router" field. Pick the ones that you wish to add.

Cheers!

**Comment by TerminalAddict** [comment]
Ah yes, I was editing the router and looking for the router group :)

Thanks for the info on updating routers
